www/src/components/guidelines/image-placeholder.js

```diff
@@ -1,19 +1,18 @@
-import React from "react"
+/** @jsx jsx */
+import { jsx } from "theme-ui"
+import { Box } from "theme-ui"
```

Comment:
Can we combine these together or does it break the JSX pragma?

```suggestion
import { jsx, Box } from "theme-ui"
```